static/figure/js/views/right_panel_view.js

```diff
@@ -243,6 +243,7 @@
         events: {
             "submit .new-label-form": "handle_new_label",
             "click .dropdown-menu a": "select_dropdown_option",
+            "click .btn-markdown": "show_markdown"
```

Comment:
Thanks @will-moore . I deliberately didn't commit figure.js, as I thought that's a generated file anyway, I'll commit it next time. Haven't looked at the PDF export yet, will do that next, have to take a look at that anyway, question in the forum about [Figure and PDF export](https://www.openmicroscopy.org/community/viewtopic.php?f=4&t=8083) .